build/template_kql/networking/networking_afd_resources_details_summary.kql

```diff
@@ -1,4 +1,4 @@
 ${extend_resource}
 | project Type, SkuName, Kind, SubscriptionId, Name, State, Location, ResourceGroup, Zones, AvailabilityZone, AvailabilitySet, Capacity, FaultDomain, OSDisk, ReadReplica, AutomaticFailover, MultipleWriteLocations, StorageAutogrow, ReplicationRole, Backup = tolower(Backup), Environment
 | where Type == 'microsoft.network/frontdoors' or Type == 'microsoft.cdn/profiles'
-| summarize ResourceTotal=count(Name), ClassicSKU=dcountif(Name, SkuName == "standard_microsoft"), NotActive=dcountif(Name, (State != "active" and State != "enabled"))
\ No newline at end of file
+| summarize ResourceTotal=count(Name), ClassicSKU=dcountif(Name, (SkuName == "standard_microsoft" or SkuName == "standard_akamai")), NotActive=dcountif(Name, (State != "active" and State != "enabled"))
```

Comment:
Sorry again. It should add "classic_frontdoor". It looks inconsistent between summary and details. 
![image](https://user-images.githubusercontent.com/3822284/228788954-905885b6-1d64-482d-92c0-e9bae7656408.png)
To be:
```kusto
| summarize ResourceTotal=count(Name), ClassicSKU=dcountif(Name, (SkuName == "standard_microsoft" or SkuName == "standard_akamai" or SkuName == "classic_frontdoor")), NotActive=dcountif(Name, (State != "active" and State != "enabled"))
```